CDCgov/prime-reportstream
# Only default xtn.2 to email when use is not set

This PR implements the test changes proposed in https://github.com/CDCgov/prime-reportstream/pull/11149 for XTN use field, line 2 of the spreadsheet. It also sets mobile numbers to have a PRS code, ensures that pager numbers and emails make it over to the other message if the use does not exist. 

Test Steps:
1. Run the tests to make sure that the fields are expected
2. Compare the spreadsheet to the tests and make sure that they align. 

## Changes
- This PR implements the test changes proposed in https://github.com/CDCgov/prime-reportstream/pull/11149 for XTN use field, line 2 of the spreadsheet. 
- Sets mobile numbers to have a PRS code
- Ensures that pager numbers and emails make it over to the other message if the use does not exist. 


## Checklist

### Testing

### Process

## Linked Issues
#11147
